frontend/alert/src/components/ManageAlert.js

```diff
@@ -0,0 +1,348 @@
+    height: ${props => props.height};
+`;
+
+const Header = ({ selected }) => {
+    const headings = ["LAST NOTIFIED", "COURSE ID", "STATUS", "REPEAT", "ACTIONS"];
+
+    return (
+        <>
+            <GridItem column="1" row="1" color="#f8f8f8" halign valign>
+                <input type="checkbox" />
+            </GridItem>
+            {!selected
+             && headings.map((heading, i) => (
+                 // eslint-disable-next-line
+                 <GridItem key={`header${i}`} column={(i + 2).toString()} row="1" color="#f8f8f8" valign>
+                     <HeaderText>{heading}</HeaderText>
+                 </GridItem>
+             ))}
+
+            {selected
+             && (
+                 <>
+                     <GridItem column="2" row="1" color="#f8f8f8" valign>
+                         <HeaderText color="#489be8">{`${selected} SELECTED`}</HeaderText>
+                     </GridItem>
+                     <GridItem column="3/7" row="1" color="#f8f8f8" valign>
+                         <RightItem>
+                             <Flex valign className={styles.headerbuttons}>
+                                 <Img src={ABell} width="0.5rem" height="0.5rem" />
+                                 <HeaderAction>RESUBSCRIBE</HeaderAction>
+                             </Flex>
+                             <Flex valign className={styles.headerbuttons}>
+                                 <Img src={XBell} width="0.5rem" height="0.5rem" />
+                                 <HeaderAction>CANCEL</HeaderAction>
+                             </Flex>
+                             <Flex valign className={styles.headerbuttons}>
+                                 <Img src={Trash} width="0.5rem" height="0.5rem" />
+                                 <HeaderAction>DELETE</HeaderAction>
+                             </Flex>
+                         </RightItem>
+                     </GridItem>
+                 </>
+             )}
+
+        </>
+    );
+};
+
+Header.propTypes = { selected: PropTypes.number };
+
+export const ManageAlertHeader = () => (
+    <Flex margin="-3.4rem 0rem 0rem 0rem">
+        <img
+            alt="Penn Course Alert logo"
+            src={Logo}
+            width="50rem"
+        />
+        <Flex col center valign halign margin="1.5rem 1.5rem 1.5rem 2rem" padding="2.2rem 0rem 0rem 0rem">
+            <Flex>
+                <Input placeholder="Course" />
+                <Button>Alert me</Button>
+            </Flex>
+            <P size="0.9rem" margin="1rem 0rem 0rem -3rem">Alert me until I cancel</P>
+        </Flex>
+    </Flex>
+);
+
+const AlertSearch = () => (
+    <SearchFlex valign>
+        <Flex valign margin="0.2rem" className={styles.search}>
+            <Img src={Search} alt="" width="0.6rem" height="0.6rem" />
+            <SearchInput type="search" placeholder="Search" />
+        </Flex>
+    </SearchFlex>
+);
+
+const AlertAction = Object.freeze({ Resubscribe: 1, Cancel: 2 });
```

Comment:
Are these enums? if so, a comment would be useful to determine that

Reply:
Fixed!